### PR TITLE
[WIP] Conntrack cleanup Fixes

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

File: pkg/datapath/conntrack.go [hunk 1]
```diff
@@ -21,26 +21,45 @@ import (
...
+	// IPv4Enabled must return true if IPv4 is enabled
+	IPv4Enabled() bool
+
+	// IPv6Enabled must return true if IPv4 is enabled
+	IPv6Enabled() bool
+}
+
+// Endpoint are the requirements of an requiring datapath plumbing
+type Endpoint interface {
+	// GetID must return the identifier of the endpoin
```
Comment (manalibhutiyani) [inline pkg/datapath/conntrack.go:47]: Nit: `endpoint`

**Comment by ianvernon** [comment]
@tgraf can this be closed?

**Comment by tgraf** [comment]
Yes!

**Pull request closed without merging**